extension.js

```diff
@@ -81,8 +81,13 @@ function _parseSearchString(searchStr) {
 function _searchContent(parsed) {
 	const items = []
 	const doc = vscode.window.activeTextEditor.document
+	const startLine = vscode.workspace.getConfiguration("swiper").get("searchFromCursor") 
+		? vscode.window.activeTextEditor.selection.active.line 
+		: 0;
+	
 	for (let i = 0; i < doc.lineCount; i++) {
-		const matches = _searchLine(i, doc.lineAt(i).text, parsed)
+		const lineIndex = (startLine + i) % doc.lineCount;
+		const matches = _searchLine(lineIndex, doc.lineAt(lineIndex).text, parsed)
```

Comment:
so the list of matches become smaller and smaller as user go down to the file? 
i prefer the behavior the matched list stay constant, but the pick.activeItems changes based on the current position of the cursor. 

thoughts?

Reply:
Hmm, you're right - that's a good idea! A bit busy these days so if you have a quick fix in mind, please go ahead and change the code. Otherwise I'll take a look soon!

Reply:
actually, i tested the current behavior is as long as you are invoking `Swiper:  Swiper Search/Resume` without changing the query string, it keeps track of the item last picked. 

Can you try again?